Bottom: 64a704f6d45dd12a8651f0dde7f71202198a0899
Top:    e43274d43766fa6827ecb7938ff67ce26b1c3f36
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-11-15 13:38:57 +0800

Refresh of tun-rx-batching

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 9c6a933..50477f5b 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1150,7 +1150,7 @@ static struct sk_buff *tun_alloc_skb(struct tun_file *tfile,
 /* Get packet from user space buffer */
 static ssize_t tun_get_user(struct tun_struct *tun, struct tun_file *tfile,
 			    void *msg_control, struct iov_iter *from,
-			    int noblock)
+			    int noblock, int more)
 {
 	struct tun_pi pi = { 0, cpu_to_be16(ETH_P_IP) };
 	struct sk_buff *skb;
